sonar-plugin-api/src/main/java/org/sonar/api/utils/System2.java

```diff
@@ -117,9 +126,11 @@ public boolean isOsWindows() {
   /**
    * True if Java 7 or Java 8 runtime environment
    * @since 4.3
+   * @deprecated in 6.4. Java 8+ is required, so this method always returns {@code true}.
    */
+  @Deprecated
   public boolean isJavaAtLeast17() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)